apps/dsp-app/src/app/user/user-form/user-form.component.ts

```diff
@@ -224,6 +198,63 @@ export class UserFormComponent implements OnInit, OnChanges {
         }
     }
 
+    editOwnAccount() {
+        this.title = this.user.username;
+        this.subtitle = "'appLabels.form.user.title.edit' | translate";
+        this.loadingData = !this.buildForm(this.user);
+    }
+
+    editExistingUser() {
+        this.title = this.name;
+        this.subtitle = "'appLabels.form.user.title.edit' | translate";
+
+        this._dspApiConnection.admin.usersEndpoint
```

Comment:
I think you should be able to get user data from the state. If not, dispatch action which loads user data and then you should be able to get it from the state. This way we get rid of unnecessary request to the server.

Reply:
Most certainly. That's exactly what is planned in the projects issue concerning the user form next week, as I wrote:. We refactor the whole component. This PR here is fixing some bugs. It is reintroducing what was there and only meant to fix, what is broken by now.

However, here we can not get the user state for this, as in this case the logged in user is editing another users account and not its own. We can not get that user via an action either, then the user which is edited will be displayed as logged in user. The goal is to not pass a user as input at all, but this would entail a proper refactoring like I am doing next week - not a bug fix. The requirements of the new user form are now defined since this morning.

If it can wait until next week I am fine.